pkg/scheduler/algorithm/predicates/csi_volume_predicate.go

```diff
@@ -134,33 +137,36 @@ func (c *CSIMaxVolumeLimitChecker) filterAttachableVolumes(
 			continue
 		}
 
-		driverName, volumeHandle := c.getCSIDriver(pvc)
+		csiNode := nodeInfo.CSINode()
+		driverName, volumeHandle := c.getCSIDriverInfo(csiNode, pvc)
```

Comment:
What if we return "", ""  but at https://github.com/kubernetes/kubernetes/pull/77595/commits/396b38f9bd2a6bd494112d4451134976bb598afc#diff-950de94621e8191fe9ca3368eef89318R208, shouldn't we log here?

Reply:
Added